cms/schemas/studentGroup.ts

```diff
@@ -5,7 +5,7 @@ import { defineField, defineType, defineArrayMember } from 'sanity';
 const GROUP_TYPES = [
     { title: 'Undergruppe', value: 'subgroup' },
     { title: 'Underorganisasjon', value: 'suborg' },
-    { title: 'Hovedstyre', value: 'board' },
+    { title: 'Hovedstyret', value: 'board' },
```

Comment:
```suggestion
    { title: 'Hovedstyre', value: 'board' },
```
skal være ubestemt form, som i "ett hovedstyre", "en undergruppe" osv...

Reply:
Åja